userguide/services.rst

```diff
@@ -2076,16 +2076,17 @@ UPS Configuration screen.
    | Driver / Remote Host    | drop-down    | For a list of supported devices, see the                                                                               |
    |                         | menu         | `Network UPS Tools compatibility list <https://networkupstools.org/stable-hcl.html>`__.                                |
    |                         |              |                                                                                                                        |
-   |                         |              | :guilabel:`Driver` becomes :guilabel:`Remote Host` when :guilabel:`UPS Mode` is set to *Slave*. The IP address of the  |
-   |                         |              | system configured as the UPS *Master* system. See this `post                                                           |
+   |                         |              | The text :guilabel:`Driver` becomes :guilabel:`Remote Host` when :guilabel:`UPS Mode` is set to *Slave*. The IP        |
+   |                         |              | address of the system configured as the UPS *Master* system. See this `post                                            |
```

Comment:
The second part is not a complete sentence: ` The IP address of the system configured as the UPS *Master* system.`